pom.xml

```diff
@@ -159,15 +158,10 @@
       </dependency>
 
       <!-- Validation used by fabric8 client-->
-      <dependency>
-        <groupId>javax.validation</groupId>
-        <artifactId>validation-api</artifactId>
-        <version>${version.validation.api}</version>
```

Comment:
I think you should clean all refs to this too see https://github.com/kiegroup/openshift-drools-hacep/pull/70/files#diff-600376dffeb79835ede4a0b285078036R34

Reply:
Yep